packages/strapi-admin/admin/src/components/Roles/temp/fakeData.js

```diff
@@ -398,34 +488,42 @@ const data = {
       actions: [
         {
           label: 'Create',
-          actionId: 'content-manager.explorer.create',
-          subjects: ['restaurant', 'address'],
-          applyToProperties: ['fields', 'locales'],
+          actionId: 'plugins::content-manager.explorer.create',
+          subjects: ['application::restaurant.restaurant', 'application::address.address'],
+          applyToProperties: [
+            'fields',
+            // Commenting the locales since the API does not support this yet
+            // 'locales'
+          ],
         },
         {
           label: 'Read',
-          actionId: 'content-manager.explorer.read',
-          subjects: ['restaurant', 'address'],
-          applyToProperties: ['fields', 'locales'],
+          actionId: 'plugins::content-manager.explorer.read',
+          subjects: ['application::restaurant.restaurant', 'application::address.address'],
+          applyToProperties: [
+            'fields',
+            // 'locales'
+          ],
         },
         {
           label: 'Update',
-          actionId: 'content-manager.explorer.update',
-          subjects: ['address', 'erestaurant'],
+          actionId: 'plugins::content-manager.explorer.update',
+          subjects: ['application::address.address', 'application::restaurant.restaurant'],
           applyToProperties: ['fields'],
         },
         {
           label: 'Delete',
-          actionId: 'content-manager.explorer.delete',
-          subjects: ['restaurant', 'address'],
+          actionId: 'plugins::content-manager.explorer.delete',
+          subjects: ['application::restaurant.restaurant', 'application::address.address'],
           applyToProperties: [],
         },
-        {
-          label: 'Publish',
-          actionId: 'content-manager.explorer.publish',
-          subjects: ['restaurant'],
-          applyToProperties: ['locales'],
-        },
+        // {
+        //   label: 'Publish',
+        //   actionId: 'plugins::content-manager.explorer.publish',
+        //   subjects: ['application::restaurant.restaurante'],
+        //   applyToProperties: [],
+        //   // applyToProperties: ['locales'],
+        // },
```

Comment:
Still need those comments ?

Reply:
Yes until we connect the view to the API, I'd rather keep the layout.